src/cascadia/Remoting/Monarch.cpp

```diff
@@ -449,7 +504,7 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
             if (auto targetPeasant{ _getPeasant(windowID) })
             {
                 auto result{ winrt::make_self<Remoting::implementation::ProposeCommandlineResult>(false) };
-
+                result->WindowName(targetWindowName);

```

Comment:
Interesting. We look up the ID from the name, and then .. tell it that the window name is the name we got the ID from?

Reply:
huh. I suppose that only really applies in the "we failed to propose the commandline, make a new window instead" case. I'll add a test